addons/hr/static/src/components/avatar_card/avatar_card_popover_patch.js

```diff
@@ -7,30 +7,42 @@ import { useService } from "@web/core/utils/hooks";
 export const patchAvatarCardPopover = {
     setup() {
         super.setup();
-        this.userInfoTemplate = "hr.avatarCardUserInfos",
+        this.userInfoTemplate = "hr.avatarCardUserInfos";
         this.actionService = useService("action");
     },
-    get fieldNames(){
-        let fields = super.fieldNames;
+    get fieldNames() {
+        const fields = super.fieldNames;
         return fields.concat([
             "work_phone",
-            "work_email", 
-            "job_title", 
-            "department_id", 
+            "work_email",
+            "name_work_location_display",
+            "type_work_location",
+            "job_title",
+            "department_id",
             this.props.recordModel ? "employee_id" : "employee_ids",
         ]);
     },
-    get email(){
+    get email() {
         return this.user.work_email || this.user.email;
     },
-    get phone(){
+    get phone() {
         return this.user.work_phone || this.user.phone;
     },
-    async onClickViewEmployee(){
+    get locationIcon() {
+        switch (this.user.type_work_location) {
+            case "office":
+                return "fa fa-fw fa-building-o";
+            case "home":
+                return "fa fa-fw fa-home";
+            default:
+                return "fa fa-fw fa-map-marker";
+        }
+    },
```

Comment:
Any particular reason to have this here rather than simple t-if in template? This feels a bit overkill.